source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -181,6 +202,31 @@ export class Carousel extends Container {
         }
     }
 
+    isCurrentPage(page: CarouselPage): boolean {
```

Comment:
you don't need to do this, you can use realIndex.

Reply:
Good call out - thanks!

Reply:
@anna-dingler,
do make sure, you've updated. I experienced the same until i cleared the web browser cache.

Reply:
FYI, https://github.com/nolimits4web/swiper/issues/1696

Reply:
@jwoo-msft I tried to refactor to use `realIndex`, but in this method, `realIndex` held the same value as `activeIndex`. 

I was able to get the correct value for `realIndex` in the event handlers though.